apm-agent-plugins/apm-apache-httpclient-plugin/src/main/java/co/elastic/apm/agent/httpclient/ApacheHttpAsyncClientInstrumentation.java

```diff
@@ -102,12 +84,14 @@ public ApacheHttpAsyncClientInstrumentation(ElasticApmTracer tracer) {
     }
 
     public static class ApacheHttpAsyncClientAdvice {
+        public static ApacheHttpAsyncClientHelper asyncHelper = new ApacheHttpAsyncClientHelper();
```

Comment:
[minor] helper instance might be `private`

Reply:
```suggestion
        private static ApacheHttpAsyncClientHelper asyncHelper = new ApacheHttpAsyncClientHelper();
```